diff --git a/src/lib/models/networks/DCNv2/dcn_v2_func.py b/src/lib/models/networks/DCNv2/dcn_v2_func.py
index 7e98f49..2c1cb85 100644
--- a/src/lib/models/networks/DCNv2/dcn_v2_func.py
+++ b/src/lib/models/networks/DCNv2/dcn_v2_func.py
@@ -6,7 +6,7 @@ from __future__ import division
 import torch
 from torch.autograd import Function
 
-from ._ext import dcn_v2 as _backend
+# from ._ext import dcn_v2 as _backend
 # from _ext import dcn_v2_double as _backend
 
 
